src/graph_compute/compute.cpp

```diff
@@ -101,6 +127,9 @@ WHEN_TEST({
     cout << "pthread workers: " << NUMA_WORKERS << '\n';
   #elif PARALLEL
     cout << "Cilk workers: " << __cilkrts_get_nworkers() << '\n';
+    #ifndef NUMA_WORKERS
```

Comment:
OK - special casing the print statement to print NUMA_WORKERS if D1_NUMA and __cilkrts_get_nworkers() if not